tests/Backend/Workspaces/WorkspacesUnloadTest.php

```diff
@@ -86,26 +101,38 @@ public function testCopyImport()
 
         $connection = $workspace['connection'];
 
-        $snowflakeConnection = new Connection([
-            'host' => $connection['host'],
-            'database' => $connection['database'],
-            'warehouse' => $connection['warehouse'],
-            'user' => $connection['user'],
-            'password' => $connection['password'],
-        ]);
-        $snowflakeConnection->query("USE SCHEMA " . $snowflakeConnection->quoteIdentifier($connection['schema']));
+        if ($connection['backend'] === parent::BACKEND_SNOWFLAKE) {
+            $db = new Connection([
+                'host' => $connection['host'],
+                'database' => $connection['database'],
+                'warehouse' => $connection['warehouse'],
+                'user' => $connection['user'],
+                'password' => $connection['password'],
+            ]);
+            // Set the session to use the workspace schema
+            $db->query("USE SCHEMA " . $db->quoteIdentifier($connection['schema']));
+
+        } else if ($connection['backend'] === parent::BACKEND_REDSHIFT) {
+            $db = new \PDO(
+                "pgsql:dbname={$connection['database']};port=5439;host=" . $connection['host'],
+                $connection['user'],
+                $connection['password']
+            );
+        } else throw new Exception("Unsupported backend for workspaces");
+
+
 
-        $snowflakeConnection->query("create table \"test.Languages3\" (
```

Comment:
@pivnicek same here. this  have to be with all these specialities in table name.